### DISCUSSION TITLE
Origin of feed tags

### DISCUSSION THREAD

**Discussion by rvrlx** [dailydotdev/daily#932] [General]
How is the available feed tag list created? Would it make sense to add the ability for the user to create his/her own feed tags? Because certain topics that I am interested in do not appear in the available tags list.

**Comment by sabdailydev** [comment]
@rvrlx Thanks for the feedback, what additional tags would you like to see?

**Reply by rvrlx** [reply]
@sabdailydev Thanks for the opportunity to provide examples. Below is a list of cybersecurity-related example tags. 

#pentest
#networksecurity
#ethicalhacking
#whitehat
#blackat
#greyhat
#cybersecurity
#bugbounty
#infosec
#redteam
#blueteam
#defensive
#offensive
#pythonhacking
#automation
#securecode
#cloudsecurity
#phishing

**Comment by idoshamun** [comment]
@rvrlx we generate our tags by using an NLP algorithm that generates them based on the content. The tags that we show are tags with a significant amount of articles in our database. The fact that the tags you suggest are missing indicates that we need more content about these topics. If you know any please suggest them. More info on how to do it: https://docs.daily.dev/docs/for-content-creators/suggest-new-source

**Reply by rvrlx** [reply]
@idoshamun OK, now I understand. Thanks. I will definitely suggest new sources as I encounter them. Thanks for the guide.
